### PR TITLE
Visualize status indicator on tasks executions

### LINKED ISSUE #1826: Add status on task execution
Depending on exit code:
0: success label
> 0: error label

### PR DESCRIPTION
**Pull request by claudiahub**
Resolves #1826 

image

### CODE REVIEW

File: ui/src/app/tasks-jobs/executions/executions.component.html [hunk 1]
```diff
@@ -88,6 +88,13 @@ <h1>{{ 'executions.main.title' | translate }}</h1>
     <ng-container>{{ 'executions.main.exitCode' | translate }}</ng-container>
     <clr-dg-filter style="display: none"></clr-dg-filter>
   </clr-dg-column>
+  <clr-dg-column
+    (clrDgColumnResize)="updateContext('sizeStatus', $event)"
+    [style.width.px]="context.sizeStatus | datagridcolumn: datagrid:contextName:15 | async"
+  >
+    <ng-container>{{ 'commons.status' | translate }}</ng-container>
```
Comment (onobc) [inline ui/src/app/tasks-jobs/executions/executions.component.html:95]: Does the ` | translate` handle i18n? (eg. `successo` when in Italian)

**Review by onobc — APPROVED** [review]
Thanks for getting this in @claudiahub . I have one question around translation but other than that, LGTM.

File: ui/src/app/tasks-jobs/executions/executions.component.html (hunk 1 above)
Comment (claudiahub) [reply ui/src/app/tasks-jobs/executions/executions.component.html:95]: @onobc  The `| translate` handles the i18n, correct. 
On this specific line ( 95 ) commons.status stands for the string equivalent to `Status` in english.
If you take a look on the data table this text should be displayed within the table header as a column indicator ( header ) for the below executions statuses ( cells ).
Let me know if this clarifies better the context, thanks !

Comment (onobc) [reply ui/src/app/tasks-jobs/executions/executions.component.html:95]: Sounds good @claudiahub , thanks for the clarification.

**Pull request merged**